integrations/http/save.go

```diff
@@ -0,0 +1,87 @@
+package http
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"go.uber.org/zap"
+)
+
+const (
+	HeaderContentType = "Content-Type"
+	ContentTypeForm   = "application/x-www-form-urlencoded"
+)
+
+// handleAuth saves a new autokitteh connection with user-submitted data.
+func (h handler) handleAuth(w http.ResponseWriter, r *http.Request) {
+	l := h.logger.With(zap.String("urlPath", r.URL.Path))
+
+	// Check "Content-Type" header.
+	ct := r.Header.Get(HeaderContentType)
+	if ct != ContentTypeForm {
+		l.Warn("Unexpected header value",
+			zap.String("header", HeaderContentType),
+			zap.String("got", ct),
+			zap.String("want", ContentTypeForm),
```

Comment:
since this is not user facing, there is no real need to print got and want

Reply:
This is related to the previous comment: the point was to have generic checks for various headers, but effectively I'm checking only Content-Type, and I always expect a web form.

So yes - added to ENG-645.